test-cases/longevity/longevity-alternator-200GB-48h.yaml

```diff
@@ -0,0 +1,33 @@
+test_duration: 2950
+prepare_write_cmd: [
+  "bin/ycsb load dynamodb -P workloads/workloada -threads 250 -p recordcount=200200300 -p fieldcount=1 -p fieldlength=1024 -p dataintegrity=true -p hdrhistogram.summary.addstatus=true", ]
+stress_cmd: [
+  "bin/ycsb run  dynamodb -P workloads/workloada -threads 250 -p recordcount=200200300 -p fieldcount=1 -p fieldlength=1024 -p operationcount=535000000 -p dataintegrity=true -p hdrhistogram.summary.addstatus=true", ]
+
+dynamodb_primarykey_type: HASH_AND_RANGE
+
+n_loaders: 3
+instance_type_db: 'i3.4xlarge'
+n_monitor_nodes: 1
+n_db_nodes: 4
+
+nemesis_class_name: 'ChaosMonkey'
+nemesis_interval: 30
+nemesis_during_prepare: false
```

Comment:
Why false? let's test with true

Reply:
@fruch , IIRC it is because the prepare phase for alternator runs in a different way as c-s.

Reply:
cause we are doing dataintegrity=true, we rather not miss anything during the prepare stage

Reply:
but it has the same effect in the prepare and not prepare, it doesn't really matter.

Reply:
we could try it,
@fabio, let give one go, if it would cause issues, we'll remove and open an trello item to try adding it back, and handle it in a different PR if needed

Reply:
will do